rabbitmq/datadog_checks/rabbitmq/data/conf.yaml.example

```diff
@@ -38,6 +38,12 @@ instances:
     #
     # tag_families: false
 
+    ## @param collect_node_metrics - boolean - optional - default: true
+    ## Node metrics are collected by default. Disable the collect_node_metrics option to skip node metric collection.
+    ## metrics pertaining to nodes
```

Comment:
Are these extra words? The sentence above this line is complete.

Reply:
Yes!